test/k8sT/manifests/server.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
 spec:
   containers:
   - name: web
-    image: jojomi/lighttpd-static
+    image: cilium/connectivity-container
```

Comment:
Please specify a version (not latest) for this image and write it here

Reply:
Fixed.